suzieq/engines/pandas/topology.py

```diff
@@ -77,7 +78,7 @@ class Services:
             extra_cols: list
             augment: any
 
-        self._namespaces = kwargs.get("namespace", self.ctxt.namespace)
+        self._namespaces = kwargs.get("namespace", [])
```

Comment:
This is the wrong fix. You'll loose the content of `self.ctxt.namespace`. You should instead put `[]` only if both `kwargs` and ctxt namespace value is empty.
This will do the trick
``` suggestion 
        self._namespaces = kwargs.get("namespace", self.ctxt.namespace) or []
```